docker/run.sh

```diff
@@ -41,4 +41,4 @@ if [[ -z "${WINTERFOX_DEDI_NODE:-}" ]]
 
 JVM_ARGS+=("$@")
 
-java -jar /app/noelware/nino/Nino.jar $JVM_ARGS
+java -jar /app/noelware/nino/bot/bin/bot $JVM_ARGS
```

Comment:
```suggestion
/app/noelware/nino/bot/bin/bot
```
JVM args get picked up by the script via the JAVA_OPTS env var

Reply:
I forgot about that, but then we should export JVM_ARGS as JVM_OPTS?

Reply:
i think the user should just set JAVA_OPTS in their compose/helm chart/whatever

Reply:
> I forgot about that, but then we should export JVM_ARGS as JVM_OPTS?

Yea, I think that would be a good idea